docs/Third_Party_Plugins.md

```diff
@@ -0,0 +1,105 @@
+# Third Party Plugins
+
+Conduit supports Third Party Plugins, but not in the way you may be used to
+in other pluggable systems. In order to limit adding dependencies, third party
+plugins are enabaled with a custom build that imports exactly the plugins you
+would like to deploy.
+
+Over time this process can be automated, but for now it is manual and requres
+writing a little bit of code.
+
+# Third Party Plugins
+
+A listing of third party plugins. If you create a plugin, please make a PR
+and add it to the list.
+
+* [conduit-plugin-template](https://github.com/algorand/conduit-plugin-template): A collection of templates.
```

Comment:
I don't see the list yet

Reply:
There is one plugin in the list :sweat_smile:

Reply:
I see conduit `list` subcommands but no actual list, even with one plugin. But I could be missing something staring right at me.

Reply:
I've attempted to clarify this section.

The bullet for `conduit-plugin-template` is the item in the list.

Reply:
I can finally see the rabbit:
![Uploading image.png…]()

Reply:
I can finally see the rabbit

<img width="1090" alt="image" src="https://github.com/algorand/conduit/assets/291133/e0b6bf5d-18c8-4b31-a41d-36b770d6cb1c">

Reply:
:duck: